index.html

```diff
@@ -0,0 +1,157 @@
+        <li><a href=""><img src="./images/github.png" alt=""></a></li>
+        <li><a href=""><img src="./images/v.png" alt=""></a></li>
+      </ul>
+    </div>
+    </section> 
+    <section class="cards">
+      <div class="card">
+        <img src="./images/1.png" alt="">
+        <p class="text-title">Tonic</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+
+      <div class="card">
+        <img src="./images/2.png" alt="">
+        <p class="text-title">Multi-post<br>stories</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+
+      <div class="card">
+        <img src="./images/3.png" alt="">
+        <p class="text-title">Tonic</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+
+      <div class="card">
+        <img src="./images/4.png" alt="">
+        <p class="text-title">Multi-post<br>stories</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+      
+
+      <section class="about-myself">
+        <div class="introduction">
+          <h1>About<br>Myself</h1>
```

Comment:
- [OPTIONAL] I would highly suggest that you only one `h1` tag in your document because it enhances your website's SEO, (search engines use `h1` tags to determine the content of the page)